lib/recording/RecordingEntrySession.js

```diff
@@ -484,6 +484,24 @@ class RecordingEntrySession {
         this.logger = loggerBase.getLogger('RecordingEntrySession', '[' + this.recordingSessionId + '] ');
         return this.playlistGenerator.setBasePath(this.recordingSessionPath);
     }
+
+    is24By7RecordingEnabled() {
+        return backendClient.getEntryInfo(this.entryObject.entryId)
+            .then((entryObj) => {
+                if (entryObj.capabilities && entryObj.capabilities.indexOf('FEATURE_KALTURA_24_BY_7_RECORDING') >= 0) {
```

Comment:
plrease rename FEATURE_KALTURA_24_BY_7_RECORDING to FEATURE_KALTURA_CONTINOUS_RECORDING

Reply:
done